enhancements/kube-apiserver/bound-sa-tokens.md

```diff
@@ -36,10 +36,23 @@ superseded-by:
 
 - Is it necessary/desirable to allow `service-account-max-token-expiration` to be configured?
 
+- If I were a customer with multiple clusters and wanted AWS IAM integration with all of
+  them, would I want to use a different issuer for each of them or reuse the same issuer?
+
+- If reusing the same issuer, would I want to share the bound token keypair across clusters
+  or supply the public keys of multiple keypairs?
+
+- Would it make sense to disable bound tokens if --service-account-issuer is not set? Or
+  should the feature be enabled by default with iss defaulted if not provided on initial
+  deployment, and it could be changed from the default subsequently if required?
+  - How does the kubelet respond to a change in issuer?
```

Comment:
Is this still open?

Reply:
Which of the two will be implemented now?

Reply:
`--service-account-issuer` will be defaulted to allow the feature to come enabled out of the box.